### PR TITLE
Venue dashboard - no design

### PR DESCRIPTION
**Pull request by mathieud**

### CODE REVIEW

File: src/pages/Account/Admin.tsx [hunk 1]
```diff
@@ -5,61 +5,192 @@ import { useUser } from "hooks/useUser";
...
+  }
+
+  return (
+    <>
+      <ul>
+        {[
+          { url: `${match.url}`, label: "Venue Infos" },
+          { url: `${match.url}/appearance`, label: "Appearance" },
+          { url: `${match.url}/events`, label: "Events" },
+        ].map((l) => (
```
Comment (antoineeudes) [inline src/pages/Account/Admin.tsx:81]: I don't like this variable name. could we name it tab?

File: src/pages/Account/Admin.tsx [hunk 2]
```diff
@@ -5,61 +5,192 @@ import { useUser } from "hooks/useUser";
...
+      where: [["owners", "array-contains", user?.uid]],
+    },
+  ]);
+
+  return (
+    <WithNavigationBar>
+      <div className="container admin-container">
+        <AuthenticationModal show={!user} onHide={() => {}} showAuth="login" />
+        <div className="title">Admin</div>
+        <div style={{ flex: "1", flexDirection: "row", display: "flex" }}>
```
Comment (antoineeudes) [inline src/pages/Account/Admin.tsx:181]: no need for `display: flex` with `flex: 1`, right?

File: src/pages/Account/Admin.tsx [hunk 3]
```diff
@@ -5,61 +5,192 @@ import { useUser } from "hooks/useUser";
...
+          <div style={{ flex: "0 0 auto", border: "red solid 1px" }}>
```
Comment (antoineeudes) [inline src/pages/Account/Admin.tsx:182]: I would prefer to see all this style in a scss file tbh

Comment (mathieud) [reply src/pages/Account/Admin.tsx:182]: css is coming in another ticket. No need to create classes I'm deleting tomorrow.

File: src/pages/Account/Admin.tsx (hunk 1 above)
Comment (mathieud) [reply src/pages/Account/Admin.tsx:81]: fixed

File: src/pages/Account/Admin.tsx (hunk 2 above)
Comment (mathieud) [reply src/pages/Account/Admin.tsx:181]: Apparently it is necessary. Doesn't respect the row flow without. Going to disappear tomorrow when we add the styles.

**Pull request merged**